main_column.py

```diff
@@ -155,38 +259,29 @@ def problem_logic_column(m):
     #         print(new_result)
 
     # D-SDA MINLP
+    # The model is built and external references are set.
     m = build_column(**model_args)
     ext_ref = {m.YB: m.intTrays, m.YR: m.intTrays}
     get_external_information(m, ext_ref, tee=globaltee)
 
+    # The model is solved using different combinations of solvers, 'k' values, and transformations. 
+    # The 'k' value is a parameter of the D-SDA method and the transformation refers to a reformulation strategy for the MINLP problem.
+    # The results are saved in a dictionary and appended to a list 'dict_data'.
     for solver in nlps:
         for k in ks:
             for transformation in transformations:
                 new_result = {}
                 m_solved, _, _ = solve_with_dsda(
-                    model_function=build_column,
-                    model_args=model_args,
-                    starting_point=starting_point,
-                    ext_dict=ext_ref,
-                    mip_transformation=True,
-                    transformation=transformation,
-                    ext_logic=problem_logic_column,
-                    k=k,
-                    provide_starting_initialization=True,
-                    feasible_model='column_' + str(NT),
-                    subproblem_solver=solver,
-                    subproblem_solver_options=nlp_opts[solver],
-                    iter_timelimit=timelimit,
-                    timelimit=timelimit,
-                    gams_output=False,
-                    tee=False,
-                    global_tee=globaltee,
+                ...
+                # A long list of arguments for the 'solve_with_dsda' function
```

Comment:
Why would you remove the options here?

Reply:
I restored the erased part of the options.